packages/fxa-auth-db-mysql/lib/db/schema/patch-114-115.sql

```diff
@@ -0,0 +1,105 @@
+--
+-- This migration introduces a table to track sent emails related to
+-- subscriptions.
+--
+
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('114');
+
+CREATE TABLE IF NOT EXISTS emailTypes (
+  id SMALLINT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  -- these are intended to be email template names
+  emailType VARCHAR(255) NOT NULL,
+  UNIQUE emailTypeIndex (emailType)
+) ENGINE=InnoDB;
+
+INSERT INTO emailTypes (emailType) VALUES
```

Comment:
Would adding a new email template require a database migration? We don't add template often but I can see this as a big barrier when we do.

Reply:
> 
> 
> Would adding a new email template require a database migration? We don't add template often but I can see this as a big barrier when we do.

I definitely don't think it's required.  It's only necessary if we want to track when that type of emails are sent.  Aside from the foreign key constraint there's nothing reinforcing the relationships among the template names, the email types in the table, and what we want to track.

Reply:
> Aside from the foreign key constraint there's nothing reinforcing the relationships among the template names

SGTM 👍🏽